tests/stubtest_third_party.py

```diff
@@ -109,19 +112,35 @@ def run_stubtest(dist: Path) -> bool:
     return True
 
 
-def run_stubtest_for(metadata: dict[str, Any], dist: Path) -> bool:
-    return has_py3_stubs(dist) and metadata.get("stubtest", True)
-
-
 # Keep this in sync with mypy_test.py
 def has_py3_stubs(dist: Path) -> bool:
     return len(glob(f"{dist}/*.pyi")) > 0 or len(glob(f"{dist}/[!@]*/__init__.pyi")) > 0
 
 
+def install_apt_packages(dist: Path, metadata: dict[str, Any], install: bool) -> bool:
+    apt_packages = metadata.get("stubtest_apt_dependencies", [])
+    if not apt_packages:
+        return True
+    if not install:
+        print(f"Ensure the following apt packages are installed for {dist.name}: {', '.join(apt_packages)}", file=sys.stderr)
+        return True
+    try:
+        apt_cmd = ["sudo", "apt", "install", *apt_packages]
```

Comment:
It would be nice to see that the command is being ran. Because of `capture_output=True`, no output ends up on terminal when it succeeds.

Also, shouldn't we pass `-y`, in case apt asks whether you want to install the package or not? It doesn't seem to happen on the CI though.

```suggestion
        apt_cmd = ["sudo", "apt", "install", "-y", *apt_packages]
        print(" ".join(apt_cmd))
```